app/timetables/tests/test_models.py

```diff
@@ -202,19 +203,32 @@ class VendorTest(TestCase):
     def setUp(self):
         self.vendor = VendorFactory()
         self.another_vendor = Vendor(
-            name='mama Taverna',
-            start_date=self.vendor.start_date,
-            end_date=self.vendor.end_date
+            name='mama Taverna'
         )
 
     def test_enforcement_of_uniqueness_of_vendor_name(self):
         self.assertRaises(ValidationError, self.another_vendor.save)
 
-    def test_enforcement_of_vendor_start_date_being_less_than_its_end_date(self):
+
+class VendorServiceTest(TestCase):
+    """Test the VendorService model"""
+
+    def setUp(self):
+        self.vendor_service = VendorServiceFactory()
+        self.another_vendor_service = VendorService(
+            timetable=self.vendor_service.timetable,
+            vendor=self.vendor_service.vendor
+        )
+
+    def test_enforcement_of_uniqueness_of_timetable_and_vendor_together(self):
+        self.assertRaises(ValidationError, self.another_vendor_service.save)
+
+    def test_enforcement_of_vendor_service_start_date_being_less_than_its_end_date(self):
         # test for start_date == end_date
-        self.another_vendor.end_date = self.vendor.start_date
-        self.assertRaises(ValidationError, self.another_vendor.save)
+        self.another_vendor_service.start_date = self.vendor_service.start_date
+        self.another_vendor_service.end_date = self.vendor_service.start_date
+        self.assertRaises(ValidationError, self.another_vendor_service.save)
```

Comment:
I think this assertion is not different from line 224 because if we take out the date alterations the test will still pass.

Can we use ValueError instead or assert that the message returned by the validationError is related to date validation.